shopinvader_delivery_instruction/README.rst

```diff
@@ -0,0 +1,51 @@
+================================
```

Comment:
Since the files in `readme` folder are empty, this readme will be wiped no? 
Maybe you forgot to commit the changes on the files?

Reply:
I create empty files because it's required to generate the readme. And the readme is required to make this PR green.
But it's generated with the OCA tool so the content is quite poor (as related files are empty).